#include <iostream>
using namespace std;

int main () {
  cout << "No exception has occured" << '\n';
  return 0;
}
